test/release/examples/primers/Makefile

```diff
@@ -69,4 +79,20 @@ FFTWlib: FFTWlib.chpl
 LAPACKlib: LAPACKlib.chpl
 	+$(CHPL) -o $@ $(LAPACK_OPTS) $<
 
+CCLIENT_OPTS = -Llib/ -linteropWithC `$CHPL_HOME/util/config/compileline --libraries`
+INTEROP_LIB_OPTS =  --library --library-makefile --savec ccode
+INTEROP_DEPS = interopWithC.chpl cHelper.h cHelper.c fact.h fact.c
+
+interopWithC: $(INTEROP_DEPS)
+	+$(CHPL) -o $@ $^
+
+
+lib/Makefile.interopWithC:
+	+$(CHPL) $(INTEROP_DEPS) $(INTEROP_LIB_OPTS)
```

Comment:
@ShreyasKhandekar : Do you know/remember why we have the Makefile build `interopWithC.chpl` in this library mode, yet don't have `start_test`, when run in this directory, do the same thing (e.g., with a `.compopts` file)?

My reason for asking is that an internal error crept in for the `make` step on multilocale platforms, yet it wasn't caught in nightly testing due to the difference.  Thanks for any context/reminders.

Reply:
Hi Brad, I looked through the history on this and I could not find a definitive reason why we don't build `interopWithC.chpl` in library mode when running `start_test`. I think that's just an oversight in testing.

Reply:
[BTW, if you want to pick this up, I'd be happy to fork it off to someone else :) and I think it fits under the umbrella of BTR week.  I opened https://github.com/chapel-lang/chapel/issues/26751 to capture the issue].

Reply:
Oh sure! I can pick this up. Could you give me more context about what you were trying to do?